tests/test_speed.py

```diff
@@ -20,6 +20,20 @@ def test_SpeedAndBufferSet(self, dvs, testlog):
         cdb = dvs.get_config_db()
         adb = dvs.get_asic_db()
 
+        # Get speed from the first port we hit in ASIC DB port walk, and
+        # assume that its the initial configured speed for all ports, and
+        # dynamic buffer profile has already been created for it.
```

Comment:
I think it's good to include this note, but I think it might also be helpful to specify why this is a valid assumption (e.g. `platform.json` guarantees `X`).

Reply:
Modified as requested.